src/App.jsx

```diff
@@ -1,11 +1,26 @@
 import * as React from 'react'
 import Button from './Button'
+import Pagination from './Pagination'
+import Header from './Header'
+import Card from './Card'
 
 export default () => {
   return (
-    <div>
-      <Button isActive>Text dazwischen</Button>
-      <Button>Text dazwischen</Button>
-    </div>
+    <section>
+      <div>
+        <Header isBackButtonVisible>Header with Button</Header>
+        <Header>Another Header</Header>
+      </div>
+      <div>
+        <Card>Card</Card>
```

Comment:
Im Card-Tag musst du noch deine in der Cards.jsx definierten Parameter und deren Attribute übergeben. Das Tag muss zudem self-closing sein.

Und dann das ganze nochmal mit "isBookmarked"